examples/basic.ts

```diff
@@ -1,8 +1,9 @@
 import {Sequelize, DataTypes} from 'sequelize';
 import { Configurations, CredentialProvider } from "@gomomento/sdk";
-import { MomentoClientGenerator } from "../src/cacheclient/momento/momento-client-generator";
-import { LoggerFactory } from "../src/logger/logger-factory";
-import {modelCacheFactory} from "../src/model-cache-factory";
+import {MomentoClientGenerator} from "@gomomento-poc/momento-sequelize-cache/src/cacheclient/momento/momento-client-generator";
+import {LoggerFactory} from "@gomomento-poc/momento-sequelize-cache/src/logger/logger-factory";
+import {modelCacheFactory} from "@gomomento-poc/momento-sequelize-cache/src/model-cache-factory";
+
```

Comment:
I will change these to conform to some best practices of exporting them from `index.ts` .. i wanted to quickly fix the current ones though first